### PR TITLE
Add proper google command 

### PR DESCRIPTION
**Pull request by JohnnyJayJay**
This pull request adds a "real" google command that displays the first google search result for a given query in chat. As a consequence, there are two new configuration settings in the dotenv file: `CSE_KEY` and `CSE_ENGINE_ID`. You can find information about how to create both of them [here](https://developers.google.com/custom-search/v1/overview) (for an engine that searches all websites you need to remove the websites you specified upon creation afterwards).

**TODO**
- [x] Testing 
- [x] Resolve conflicts with the `lmgtfy` command, which currently shares two aliases with the `google` command. Maybe the former becomes obsolete and can be deleted (it's not as helpful anyway)? Otherwise we need to think of distinct names.

### CODE REVIEW

File: src/main/kotlin/com/github/seliba/devcordbot/commands/general/GoogleCommand.kt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            val result = results[0]
+            context.respond(
+                """
+                **${result.title}**
+                *${result.snippet}*
+                ${result.link}""".trimIndent()
+            )
+        }
+    }
+}
```
Comment (DRSchlaubi) [inline src/main/kotlin/com/github/seliba/devcordbot/commands/general/GoogleCommand.kt]: _ guy won't like this

Comment (JohnnyJayJay) [reply src/main/kotlin/com/github/seliba/devcordbot/commands/general/GoogleCommand.kt]: What, curly braces? Should have picked a different language then

Comment (DRSchlaubi) [reply src/main/kotlin/com/github/seliba/devcordbot/commands/general/GoogleCommand.kt]: No there is no new line at the end of the file

**Review by DRSchlaubi — APPROVED** [review]
Have fun creating the key kobold

File: src/main/kotlin/com/github/seliba/devcordbot/commands/general/GoogleCommand.kt (hunk 1 above)
Comment (JohnnyJayJay) [reply src/main/kotlin/com/github/seliba/devcordbot/commands/general/GoogleCommand.kt]: fixed

File: .env.example [hunk 2]
```diff
@@ -13,3 +13,6 @@ AUTO_HELP_WHITELIST="comma,seperted,list,of,channel,ids"
 AUTO_HELP_BLACKLIST="comma,seperted,list,of,channel,ids"
 AUTO_HELP_KNOWN_LANGUAGES="java,go"
 BOT_OWNERS="123456,654321"
+# Google CSE (https://developers.google.com/custom-search/v1/overview)
+CSE_KEY="API_KEY"
+CSE_ENGINE_ID="21j34jk5213j12"
```
Comment (jkoenig134) [inline .env.example]: -> CSE_ID

**Review by jkoenig134 — CHANGES_REQUESTED** [review]
Ich bekomme noch eine Warnung von der CSE api
`Mai 15, 2020 6:32:23 PM com.google.api.client.googleapis.services.AbstractGoogleClient <init>
WARNUNG: Application name is not set. Call Builder#setApplicationName.`
Kannst du das noch beheben?

File: src/main/kotlin/com/github/seliba/devcordbot/commands/general/GoogleCommand.kt [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import com.github.seliba.devcordbot.command.permission.Permission
+import com.github.seliba.devcordbot.constants.Embeds
+import com.github.seliba.devcordbot.menu.Paginator
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
+import com.google.api.client.json.jackson2.JacksonFactory
+import com.google.api.services.customsearch.Customsearch
+
+class GoogleCommand(private val apiKey: String, private val engineId: String) : AbstractCommand() {
+
+    private val search = Customsearch(GoogleNetHttpTransport.newTrustedTransport(), JacksonFactory(), null)
```
Comment (jkoenig134) [inline src/main/kotlin/com/github/seliba/devcordbot/commands/general/GoogleCommand.kt]: Use builder to set application name.

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**